migrator/migrations/m_191_to_m_192_reoprt_config/migration_impl.go

```diff
@@ -0,0 +1,121 @@
+package m191tom192
+
+import (
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/storage"
+	updatedSchema "github.com/stackrox/rox/migrator/migrations/m_191_to_m_192_reoprt_config/schema"
+	"github.com/stackrox/rox/migrator/types"
+	"github.com/stackrox/rox/pkg/postgres/pgutils"
+	"github.com/stackrox/rox/pkg/utils"
+	"github.com/stackrox/rox/pkg/uuid"
+	"gorm.io/gorm/clause"
+)
+
+func migrate(database *types.Databases) error {
+	_ = database // TODO(dont-merge): remove this line, it is there to make the compiler happy while the migration code is being written.
+	// Use databases.DBCtx to take advantage of the transaction wrapping present in the migration initiator
+	db := database.GormDB
+	pgutils.CreateTableFromModel(database.DBCtx, db, updatedSchema.CreateTableReportConfigurationsStmt)
+	db = db.WithContext(database.DBCtx).Table("report_configurations")
+
+	dbSnapshot := database.GormDB
+	pgutils.CreateTableFromModel(database.DBCtx, dbSnapshot, updatedSchema.CreateTableReportSnapshotsStmt)
+	dbSnapshot = dbSnapshot.WithContext(database.DBCtx).Table("report_snapshots")
+
+	rows, err := db.Rows()
+	if err != nil {
+		return errors.Wrapf(err, "failed to iterate table %s", "report_configurations")
+	}
+	if rows.Err() != nil {
+		utils.Should(rows.Err())
+		return errors.Wrapf(rows.Err(), "failed to get rows for %s", "report_configurations")
+	}
+	defer func() { _ = rows.Close() }()
+	var convertedReportConfigs []*updatedSchema.ReportConfigurations
+	var reportSnapshots []*updatedSchema.ReportSnapshots
+	for rows.Next() {
+		var reportConfig *updatedSchema.ReportConfigurations
+		if err = db.ScanRows(rows, &reportConfig); err != nil {
+			return errors.Wrap(err, "failed to scan rows")
+		}
+
+		reportConfigProto, err := updatedSchema.ConvertReportConfigurationToProto(reportConfig)
+		if err != nil {
+			return errors.Wrapf(err, "failed to convert %+v to proto", reportConfig)
+		}
+		convertedReportConfigs = append(convertedReportConfigs, reportConfig)
+		if reportConfigProto.Version != 0 {
+			continue
+		}
+		newConfig := reportConfigProto.Clone()
+		newConfig.Id = uuid.NewV4().String()
+
+		newConfig.ResourceScope = &storage.ResourceScope{
+			ScopeReference: &storage.ResourceScope_CollectionId{CollectionId: reportConfigProto.GetScopeId()},
+		}
+		if reportConfigProto.GetVulnReportFilters().GetSinceLastReport() {
+			newConfig.Filter = &storage.ReportConfiguration_VulnReportFilters{
+				VulnReportFilters: &storage.VulnerabilityReportFilters{
+					CvesSince: &storage.VulnerabilityReportFilters_SinceLastSentScheduledReport{
+						SinceLastSentScheduledReport: true,
+					},
+				}}
+		} else {
+			newConfig.Filter = &storage.ReportConfiguration_VulnReportFilters{
+				VulnReportFilters: &storage.VulnerabilityReportFilters{
+					CvesSince: &storage.VulnerabilityReportFilters_AllVuln{
+						AllVuln: true,
+					},
+				},
+			}
+		}
+
+		newConfig.Version = 1
```

Comment:
Is this supposed to be 2 or 1? In [convert.go](https://github.com/stackrox/stackrox/pull/7866/files#diff-0d0ec0f87ec4ff22adfc75c19e18f5eebf9efbc1deab1ae7f1048dacf9e83db4R60) you set it to 2